pulse/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '9.0'
+
+target 'pulse' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for pulse
+  pod 'SwiftyJSON'
+  pod 'Alamofire'
+end
```

Comment:
Checking in the `Podfile` and `Podfile.lock`. Quick resource as to why we should do that https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control

^^ Will add this link to Trello as well